app/src/main/java/space/taran/arknavigator/mvp/model/repo/index/ResourceMeta.kt

```diff
@@ -1,22 +1,15 @@
 package space.taran.arknavigator.mvp.model.repo.index
 
+import space.taran.arklib.index.ResourceMeta
 import space.taran.arknavigator.mvp.model.dao.ResourceWithExtra
 import space.taran.arknavigator.mvp.model.repo.kind.GeneralKindFactory
-import space.taran.arknavigator.mvp.model.repo.kind.ResourceKind
 import space.taran.arknavigator.utils.extension
 import java.nio.file.Files
 import java.nio.file.Path
 import java.nio.file.attribute.FileTime
+typealias ResourceMeta = ResourceMeta
 
-data class ResourceMeta(
-    val id: ResourceId,
-    val name: String,
-    val extension: String,
-    val modified: FileTime,
-    val size: Long,
-    val kind: ResourceKind?
-) {
-
+class ResourceMetaExtra {
```

Comment:
This name is confusing. Should be replaced with just normal static methods/object.
"Extra" is only untyped data in `ResourceMeta`. See `ResourceWithExtra` class.